test/test_mbtiler.py

```diff
@@ -60,7 +60,7 @@ def test_webp_writer():
 
     test_bytearray = _encode_as_webp(test_data)
 
-    assert len(test_bytearray) == 42
+    assert len(test_bytearray) == 34
```

Comment:
@sgillies @dnomadb Not really sure why this changed!

Reply:
I don't know. I took myself off the review list for this one because I gotta focus on something else and I haven't used this project yet.